File: public/phase-2-20160620022203/PYTHON/apartment.py
#!/usr/bin/python
# Name : Apartment
# input:
# n :  the size of Shalkan’s apartment plan
# m : the size of Shalkan’s apartment plan
# nm :  the description of the apartment plan
elements = raw_input().strip().split(' ')
n = int(elements[0])
m = int(elements[1])
nm = []
for _ in xrange(n):
	nm_elements = raw_input().strip().split(' ')
	nm.append(nm_elements)


# write your code here
# store your results in `new_plan`

# output

# Dummy Data
new_plan = [['.','*','.','*','.'],['*','*','*','*','*'],['.','*','.','*','.'],['*','*','*','*','*'],['.','*','.','*','.']]

for idx in xrange(n):
	print(" ".join(str(e) for e in new_plan[idx]))

File: public/phase-2-20160620022203/JAVASCRIPT/apartment.js
// Name : Apartment
// input:
// n :  the size of Shalkan’s apartment plan
// m : the size of Shalkan’s apartment plan
// nm :  the description of the apartment plan
var idx,new_plan;
var elements = readline().split(" ");
var n = parseInt(elements[0], 10);
var m = parseInt(elements[1], 10);
var nm = new Array([]);
var nm_elements = "";
for(idx=0;idx<n;idx++){
	nm_elements = readline().split(' ');
	nm[idx] = new Array([]);
	for(jdx=0;jdx<m;jdx++){
		nm[idx][jdx] = nm_elements[jdx];
	}
}


// write your code here
// store your results in `new_plan`

// Dummy Data
new_plan = [['.','*','.','*','.'],['*','*','*','*','*'],['.','*','.','*','.'],['*','*','*','*','*'],['.','*','.','*','.']];


// output
for(idx=0;idx<n;idx++){
	print(new_plan[idx].join(' '));
}
